.github/workflows/post-commit.yml

```
@@ -100,6 +106,50 @@ jobs:
+      # The comment also answers the question the run cannot: is this refusal
+      # enforced here? A private repository in a Free-plan org gets no ruleset,
+      # so the red status stops nothing and saying so plainly is the only
+      # honest thing left. One comment per pull request, rewritten in place.
+      - name: post the verdict on the pull request
+        env:
+          GH_TOKEN: ${{ github.token }}
+          PR: ${{ github.event.pull_request.number }}
+          REPO: ${{ github.repository }}
+          REPORT: ${{ needs['post-commit'].outputs.report }}
+          RUN_URL: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2016:info:10:17: Expressions don't expand in single quotes, use double quotes for that

(shellcheck)